src/hypergraph/viz/renderer/edges.py

```
@@ -0,0 +1,457 @@
+                        rf_edge["data"]["label"] = "True"
+                    elif target == branch_data["when_false"]:
+                        rf_edge["data"]["label"] = "False"
+
+            edges.append(rf_edge)
+
+
+def compute_edges_for_state(
+    flat_graph: nx.DiGraph,
+    expansion_state: dict[str, bool],
+    input_spec: dict[str, Any],
+    show_types: bool,
```

Ruff (0.14.14)

[warning] Unused function argument: `show_types`

(ARG001)